modules/frontend/service/frontend.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=frontend service
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/Niffler/modules/frontend/service/frontend_service.sh
+TimeoutStartSec=360
+
+[Install]
+WantedBy=default.target
```

Comment:
Can you pls rename this file as niffler.service instead?

Otherwise, from the operating system level, when someone lists the running services, they will see a "frontend.service" and be puzzled what it could be (and a possibility of naming clashes too, if there could be other frontend services).